src/indexes.rs

```diff
@@ -11,7 +11,7 @@ use serde::{de::DeserializeOwned, Deserialize, Serialize};
 use std::{collections::HashMap, fmt::Display, time::Duration};
 use time::OffsetDateTime;
 
-/// An index containing [Document]s.
+/// An index containing \[Document\]s.
```

Comment:
```suggestion
/// A Meilisearch [index](https://docs.meilisearch.com/learn/core_concepts/indexes.html).
```

Maybe we are not helping by mentioning documents here. Technically, an index could also not contain documents. What do you think of this suggestion?

Reply:
I think it's fine. :D